### PR TITLE
Adding post about afCa and the new analysis capabilities

### PR DESCRIPTION
**Pull request by oddcoder**

### CODE REVIEW

File: content/posts/Improving-analysis.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
++++
+date = "2016-5-9"
+draft = false
+title = "Improving analysis"
+slug = "Improving-analysis"
+aliases = [
+	"Improving-analysis"
+]
++++
+One of the main tasks of Radare2 is to do statically analyse executables. This include binary files disassembly, analysing functions setting [calling conventions](https://en.wikipedia.org/wiki/Calling_convention), auto detecting arguments and type propagation.
```
Comment (radare) [inline content/posts/Improving-analysis.md]: feel free to add a paragraph to present yourself and explain your that this is part of your task in the gsoc

File: content/posts/Improving-analysis.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+New analysis round is added for argument detection. It is architecture independent and supposed to capture all arguements and variables then auto rename them. This analysis round is built on top of [ESIL](https://github.com/radare/radare2book/blob/master/esil.md). It will detect all the `base pointer + num` and store them as arguments and `base pointer - num` and store them as variable. The `stack pointer + num` will always be stored as argument whether it is argument or variable. Identifying the final state of is that `stack pointer offset` is argument or variable is still work in progress
+
+`Radare2` also supports renaming declared variable/arguments this can be done using the command `afXn`, where `X` can be:
```
Comment (radare) [inline content/posts/Improving-analysis.md]: add a example with small function disasm with function renaming and such

Comment (Maijin) [inline content/posts/Improving-analysis.md]: arguements => arguments

**Comment by Maijin** [comment]
Any pictures, asciinema?

**Pull request merged**